server/service/apple_mdm_cmd_results.go

```diff
@@ -0,0 +1,55 @@
+package service
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/fleetdm/fleet/v4/server/contexts/ctxerr"
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	apple_mdm "github.com/fleetdm/fleet/v4/server/mdm/apple"
+	"github.com/fleetdm/fleet/v4/server/mdm/nanomdm/mdm"
+	kitlog "github.com/go-kit/log"
+	"github.com/micromdm/plist"
+)
+
+func NewDeviceLocationResult(result *mdm.CommandResults, hostID uint) (DeviceLocationResult, error) {
+	var x deviceLocationResult
```

Comment:
Any reasonw e couldn't unmarshal directly into this struct, and init the struct with hostID?

Reply:
I don't feel super strongly on this, so fair enough.

I do think we can init the returned object with the host ID though at lease.